nemo/collections/asr/models/asr_model.py

```diff
@@ -47,3 +51,61 @@ def multi_test_epoch_end(self, outputs, dataloader_idx: int = 0):
         wer_denom = torch.stack([x['test_wer_denom'] for x in outputs]).sum()
         tensorboard_logs = {'test_loss': val_loss_mean, 'test_wer': wer_num / wer_denom}
         return {'test_loss': val_loss_mean, 'log': tensorboard_logs}
+
+    def export(
+        self,
```

Comment:
This will work - as long as the model has just encoder - decoder architecture. As this is the base class of all ASR Models - including RNNT models - this assumption is invalid. However, RNNT is under development so could you override the export() method in both RNNT model classes and explicitly raise a NotImplementedError()?

Reply:
In fact, it can be more cleanly managed by making this ASR export method be in a mixin class - one for `ExportableEncDec` (which has this method in it) and another `ExportableRNNT` (which raises NotImplemented for now).

After that, let all the CTC and classification / label models extend the `ExportableEncDec` class which gives them proper export, and RNNT extend the `ExportableRNNT` which raises an error during export (for now).

This is infact how we add Subword support to ASR Models - https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/parts/mixins.py